packages/common/src/versioning/_internal/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from "./constants";
```

Comment:
I _think_ our recent import wars suggested that having this file, in `_internal` is not a good idea - we want to import directly from files vs a `/index`